static/src/stylesheets/module/nav/_brand-bar.scss

```diff
@@ -43,6 +43,24 @@
         margin-left: $gs-gutter / 2;
     }
 }
+.brand-bar__item .brand-bar__item--split--first {
+    margin-left: $gs-gutter/4;
+    height: 34px;
+}
+.brand-bar__item--split--first::after {
+    content: '\200B'; //zero width space
+    border-right: 1px solid rgba(255, 255, 255, .2);
+    left: 11px;
+    position: relative;
+    height: 24px;
+    bottom: -5px;
+    display: block;
+}
+
+.brand-bar__item--split--last {
+    padding-left: 11px;
+    height: 34px;
```

Comment:
Is it possible to use less magic numbers for these heights, positions and paddings? Perhaps some multiples of `$gs-gutter` (for horizontal spacing) `$gs-baseline` (for vertical spacing) and `$gs-height` (for element height)